frontend/components/bc/toggle/MultiBarButton.vue

```diff
@@ -17,16 +19,33 @@ const icon = computed(() => {
 </script>
 
 <template>
-  <ToggleButton v-model="selected" class="bc-toggle" on-label="''" off-icon="''">
-    <template #icon="slotProps">
-      <slot name="icon" v-bind="slotProps">
-        <FontAwesomeIcon v-if="icon" :icon="icon" />
-      </slot>
+  <BcTooltip>
+    <template #tooltip>
+      <div class="button-tooltip">
+        <div>{{ selected ? $t('filter.enabled'): $t('filter.disabled') }}</div>
+        <div v-if="tooltip" class="individual">
+          {{ tooltip }}
```

Comment:
**Request:** I know it is designed like this, but please ask the product owner/designer if the order of these two texts can be changed. Reading "If this is enabled, it does X" BEFORE "Filter is enabled" flows better I believe.

Reply:
changed